base-image/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o errexit
+
+BUILD_DIR='/home/builder/tmp'
+
+mkdir -p $BUILD_DIR
+cp -r $SOURCE_DIR/* $BUILD_DIR/
+cd $BUILD_DIR
+source poky/oe-init-build-env build
+bitbake -k core-image-minimal
+cd tmp/deploy/images/
+cp $(find . -name *.rootfs.tar.bz2) $DEST_DIR/rootfs.tar.bz2
```

Comment:
I think this way will work out of the box since there is only one file that its name ends with `.rootfs.tar.bz2` and we just need to find it.
We can predict the filename but this script will become way more complicated than it should be since rootfs files are inside subdirectory of `deploy/images` (qemuXXX, .i.e qemux86-64 for amd64 iirc). So we need to include that qemu thing in the build script to predict the name and path to rootfs file...